_includes/components/hero.html

```diff
@@ -1,4 +1,4 @@
-<div class="hero">
+<div class="hero project-section dark">
```

Comment:
so first, the hero needs to be considered a project section or it'll always break on the hero

Reply:
:0